src/models/methods/channelAuthorize.ts

```diff
@@ -0,0 +1,18 @@
+import {BaseRequest, BaseResponse} from './baseMethod'
+
+export interface ChannelAuthorizeRequest extends BaseRequest {
```

Comment:
Actually, I think we don't want to implement this at all (for the same reason we're not implementing `Sign` and `SignFor`, because it requires sending your secret over the network to a node that you likely don't have control of - that's not safe, and we don't want to encourage that).

Reply:
Oh I see, I'll delete this method then.